pkg/controller/gcppubsub/receive_adapter/receive_adapter.go

```diff
@@ -0,0 +1,110 @@
+package main
```

Comment:
this should move to a `cmd/gcppubsub_recieve_adapter` dir (or similar)

Reply:
:+1: That's consistent with the kubernetes event source receive adapter location. But spell `receive` correctly. :grin:

Reply:
Done.